packages/web-components/src/components/ic-card/ic-card.stories.mdx

```diff
@@ -836,3 +836,183 @@ import Button from "../ic-button/readme.md";
     ></a>`}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  clickable: false,
+  disabled: false,
+  expandable: false,
+  fullWidth: false,
+  heading: "Heading",
+  href: "/",
+  message: "Message",
+  subheading: "Subheading",
+  iconSlot: "icon",
+  badgeSlot: "badge",
+  imageTopSlot: "image-top",
+  imageMidSlot: "image-mid",
+  interactionButtonSlot: "interaction-button",
+  interactionControlsSlot: "interaction-controls",
+  adornmentSlot: "adornment",
+  expandedContentSlot: "expanded-content",
```

Comment:
I don't have the option to remove the slots I don't want to see. Is it possible to integrate this?

Reply:
It should be possible to remove every slot using the select dropdown and selecting the empty option

Reply:
I'm not sure if that's possible, as it's currently either using the slot name or using an empty string to determine if the slot content shows or not (since it won't let me do conditional rendering)

Reply:
I've updated to use a radio control for slots so it hopefully makes it more obvious that an empty string is an option 👍

Reply:
Awesome, I'll check it out once it's deployed. Looked into it a bit more and found that Storybook allows you to [map options to other values](https://storybook.js.org/docs/api/arg-types#mapping)

Reply:
Nice find! I've updated it to use a boolean value and use mapping